addon/components/frost-table-header.js

```diff
@@ -62,7 +62,7 @@ export default Component.extend({
   @computed('columns')
   _hasCategories (columns) {
```

Comment:
Another repeated computed property. Although I'm liking this implementation better since `some` is more elegant than `reduce`.